app/models/concerns/smooch_messages.rb

```diff
@@ -291,7 +291,8 @@ def bundle_list_of_messages_to_items(list, last)
         if message['type'] == 'text'
           # Get an item for long text (message that match number of words condition)
           if message['payload'].nil?
-            messages << message if ::Bot::Alegre.get_number_of_words(message['text'].to_s) > CheckConfig.get('min_number_of_words_for_tipline_submit_shortcut', 10, :integer)
+            contain_link = self.extract_url(message['text'])
+            messages << message if !contain_link.nil? || ::Bot::Alegre.get_number_of_words(message['text'].to_s) > CheckConfig.get('min_number_of_words_for_tipline_submit_shortcut', 10, :integer)
```

Comment:
For consistency, please just rename `contain_link` to `contains_link`.

Reply:
Done